content/en/docs/ops/configuration/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,134 @@
+When referring to the TCP protocol in Istio contents this typically means we are treating it as an opaque stream of bytes,
+and not parsing higher level constructs such as TLS or HTTP.
+
+## Traffic Routing
+
+When the Istio proxy receives a request, it must decide where to forward the request to, if anywhere.
+By default, this will be the original service that was request, unless [customized](/tasks/traffic-management/traffic-shifting/).
+How this works depends on the protocol used.
+
+### TCP
+
+When processing TCP traffic, Istio has a very small amount of useful information to route the connection based on - only the destination IP and Port.
+Fortunately, this is enough to determine the intended Service; the proxy is configured to listener on each service IP:Port pair and forward traffic to that service.
+
+For customizations, a TCP `VirtualService` can be configured, which allows [matching on specific IPs and ports](/reference/config/networking/virtual-service/#L4MatchAttributes).
+
+### TLS
+
+When processing TLS traffic, Istio has slightly more information available than raw TCP: we can inspect the [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication).
+
+For standard Services, the same IP:Port matching as raw TCP is used.
+However, for services that do not have a Service IP defined, such as [ExternalName services](#externalname-services), the SNI will be used for routing.
+
+Additionally, custom routing can be configured with a TLS `VirtualService` to [matching on SNI](/config/networking/virtual-service/#TLSMatchAttributes).
+
+### HTTP
+
+HTTP allows much richer routing than TCP and TLS. With HTTP, we are able to route individual HTTP request, rather than just connections.
+In addition, a [number of different request attributes](/config/networking/virtual-service/#HTTPMatchRequest) are available, such as host, path, headers, query parameters, etc.
+
+While TCP and TLS traffic generally behave the same with or without Istio (assuming no configuration has been applied to customize the routing), HTTP has significant differences.
+
+* Istio will load balance individual requests. In general, this is highly desirably, especially in scenarios with long-lived connections such as gRPC and HTTP/2, where connection level load balancing is ineffectively.
+* Requests are routed based on the port and *`Host` header*, rather than port and IP. This means the destination IP address is effectively ignored. For example, `curl 8.8.8.8 -H "Host: productpage.default.svc.cluster.local"`, would be routed to the `productpage` Service.
+
+## Unmatched traffic
+
+If traffic cannot be matched using one of the methods described above, it is treated as [passthrough traffic](/docs/tasks/traffic-management/egress/egress-control/#envoy-passthrough-to-external-services).
+By default, these requests will be forwarded as-ism, which ensures that traffic to services that Istio is not aware of, such as external services that do not have `ServiceEntry`s created.
```

Comment:
as-is

Reply:
Sentence just seems to end. Maybe ... as-is, ensuring traffic to services that Istio is not aware of such as ..